manifests/admin.pp

```diff
@@ -93,14 +93,14 @@
   $enable_color       = $pulp::admin::params::enable_color,
   $wrap_to_terminal   = $pulp::admin::params::wrap_to_terminal,
   $wrap_width         = $pulp::admin::params::wrap_width,
-  $puppet             = $pulp::admin::params::puppet,
-  $docker             = $pulp::admin::params::docker,
-  $nodes              = $pulp::admin::params::nodes,
-  $python             = $pulp::admin::params::python,
-  $rpm                = $pulp::admin::params::rpm,
+  $enable_puppet      = $pulp::admin::params::enable_puppet,
+  $enable_docker      = $pulp::admin::params::enable_docker,
+  $enable_nodes       = $pulp::admin::params::enable_nodes,
+  $enable_python      = $pulp::admin::params::enable_python,
+  $enable_rpm         = $pulp::admin::params::enable_rpm,
```

Comment:
Not a requirement for the PR, but you cold consider adding `validate_bool` for each to enforce the type